README.md

```diff
@@ -1,58 +1,56 @@
 # Micromasters
 Portal for learners and course teams to access MITx Micromasters programs
 
-## Getting Started
+## Major Dependencies
+- pip, virtualenv, homebrew (OSX only)
+- Postgres >= 9.4
+  - OSX recommended install method: [Postgres.app](http://postgresapp.com/)
+- Python 3
+  - OSX recommended install method: Homebrew `brew install python3`
+- Docker (_optional, but strongly recommended_)
```

Comment:
It seems odd that we aren't saying Docker is required here but the rest of our instructions are how to set this up with Docker.